server/tests/endpoints/certification/test_certification.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 Canonical Ltd.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Written by:
+#        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
+
+
+from datetime import timedelta, datetime
+from fastapi.testclient import TestClient
+
+
+def test_certified_status(test_client: TestClient):
+    response = test_client.post(
+        "/v1/certification/status",
+        json={"vendor": "Dell", "model": "ChengMing 3980 (i3-9100)"},
+    )
+
+    assert response.status_code == 200
+    expected_response = {
+        "status": "Certified",
```

Comment:
how does this pass? I don't see any test data being inserted into the db. Perhaps it uses the seed data script? if so that would be coupling test cases to seed data. It would be better if the test cases were standalone and populate whatever data they need. Seed data should only run in dev database not test one. For tests we probably need to create a new test db for all of the tests, run db migrations on it and then for each test case we need to make sure to empty it out during the cleanup. Setting this up can be hairy so I'd recommend you look at how TO is doing it